corehq/apps/hqwebapp/migrations/0002_auto_20170416_2117.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.6 on 2017-04-16 21:17
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hqwebapp', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='HashedPasswordLoginAttempt',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('username', models.CharField(db_index=True, max_length=255)),
+                ('password_hash', models.CharField(db_index=True, max_length=255)),
```

Comment:
I think just remove the migrations - we can clean up staging by manually dropping the tables.